### ISSUE TITLE
Dice validation score is too low

### ISSUE THREAD

**Issue by hwjung92** [qubvel/segmentation_models.pytorch#678]
The pretrained_model provided was loaded and experimented.
It should show more than 95% accuracy
However, I got this result image. 

![image](https://user-images.githubusercontent.com/7631380/197712185-edd72449-40a2-406d-901d-ae33964485c7.png)

Dice validation score was around 0.65.
I am curious of the pytorch version (torch==1.12.0+cu113, torchvision==0.13.0+cu113) at docker.
Do you have similar issues?

**Comment by hwjung92** [comment]
It was related to efficientnet-pytorch.
efficientnet-pytorch==0.6.3 (working)
efficientnet-pytorch==0.7.1 (not working)
